Automattic/wp-calypso
# Cards and Forms: Update headings and labels to use sentence case

#### Changes proposed in this Pull Request

This PR updates the following to use sentence case:

- Card headings
- Form labels

> Page titles remain in title case.
All other titles can be in sentence case, except when referring to an official/branded feature or service name.
Taglines, subheadings, descriptions, etc. should always end with a period (and, ideally, be actual sentences rather than fragments).

For more info: p2-pbAPfg-dH

# **Before (card heading):**
![Screen Shot 2020-04-29 at 2 54 12 PM](https://user-images.githubusercontent.com/4924246/80650762-7043b880-8a29-11ea-9c9e-dad6349fb9fd.png)

# **After (card heading):**
![Screen Shot 2020-04-29 at 2 54 08 PM](https://user-images.githubusercontent.com/4924246/80650773-75a10300-8a29-11ea-8528-6a0d675998fa.png)

# Before (form label):
![Screen Shot 2020-04-29 at 2 58 18 PM](https://user-images.githubusercontent.com/4924246/80650985-e2b49880-8a29-11ea-9443-6f30ed7d65ab.png)

# After (form label):
![Screen Shot 2020-04-29 at 2 58 23 PM](https://user-images.githubusercontent.com/4924246/80650992-e6481f80-8a29-11ea-966c-943638ee6fb0.png)

# Before (/plans):
![Screen Shot 2020-05-01 at 2 38 02 PM](https://user-images.githubusercontent.com/4924246/80843544-6f419100-8bb9-11ea-820a-bbe146eae45e.png)

# After (/plans):
![Screen Shot 2020-05-01 at 2 38 20 PM](https://user-images.githubusercontent.com/4924246/80843552-749edb80-8bb9-11ea-9137-206188bcd537.png)

#### Testing instructions

- Checkout this branch locally or visit https://calypso.live/?branch=card-headings-form-labels-sentence-case
- Go through each section looking and make sure card headings and form labels are written in sentence case
- Try with different site types (Simple, Atomic, Jetpack), different plans (free, paid), or any other combination you can think of
- Please note that some pages in checkout will remain in title case. They will need to be updated in the backend. See: https://github.com/Automattic/wp-calypso/issues/41768

![Screen Shot 2020-05-01 at 11 44 54 AM](https://user-images.githubusercontent.com/4924246/81015894-54b72400-8e14-11ea-9c81-20835c98be2e.png)

**Sections:**
- [x] My Home
- [x] Stats - Fixes: https://github.com/Automattic/wp-calypso/issues/40622
- [x] Plan
- [x] Import
- [x] Export
- [x] Marketing
- [x] Activity
- [x] Hosting configuration
- [x] Domains
- [x] People

**/me**
- [x] My profile
- [x] Account settings
- [x] Manage purchase
- [x] Privacy
- [x] Notification settings
- [x] Blocked sites

cc @rickybanister @lcollette @gwwar @kwight
